src/app/lib/utilities/getMostWatchedUrl/index.js

```diff
@@ -1,6 +1,14 @@
-const getMostWatchedEndpoint = ({ service, variant }) =>
+const getBaseUrl = env =>
+  ({
+    live: 'https://www.bbc.com/',
+    test: 'https://test.bbc.com/',
+  }[env] || '/');
+
+const getExtension = env => (env !== 'local' ? '.json' : '');
```

Comment:
Why are we taking the .json extension out on local?

Reply:
If we don't provide a fully resolved URL to `fetchPageData` it tries to work it out for us (using its `getUrl` function) - that includes adding in the `.json` extension - https://github.com/bbc/simorgh/pull/7954/files#diff-93bc8a6ae91fb1ee01233d35e3590c4dR32

Reply:
Ah, yes.